python/ray/tests/test_concurrency_group.py

```diff
@@ -180,6 +180,47 @@ def ping(self):
     print(ray.get(n.ping.remote()))
 
 
+@ray.remote(concurrency_groups={"io": 1, "compute": 1})
+class Actor:
+    def __init__(self):
+        self.threading_data = threading.local()
+
+    def set(self, value):
+        self.threading_data.value = value
+        return threading.current_thread().ident
+
+    def get(self):
+        return self.threading_data.value, threading.current_thread().ident
```

Comment:
```suggestion
        self._thread_local_data = threading.local()

    def set(self, value):
        self._thread_local_data.value = value
        return threading.current_thread().ident

    def get(self):
        return self._thread_local_data.value, threading.current_thread().ident
```

please also add type hints

Reply:
Updated: https://github.com/ray-project/ray/pull/50644/commits/ba4b1521cc6509405c8404a1eafea43111146118